src/env.h

```diff
@@ -703,6 +709,7 @@ class Environment {
   std::unordered_map<uint32_t, loader::ModuleWrap*> id_to_module_map;
   std::unordered_map<uint32_t, contextify::ContextifyScript*>
       id_to_script_map;
+  std::set<CompileFnEntry*> compile_fn_entries;
```

Comment:
std::unordered_set? We are likely to create a lot of entries in a process, so that might be more performant?

Reply:
Good point, will change.